tests/test_find_bugs_sweep_cli.py

```diff
@@ -210,11 +210,11 @@ class TestCategorizeBugsByType(unittest.TestCase):
     def test_categorize_bugs_by_type(self):
         advisory_id_map = {'image': 1, 'rpm': 2, 'extras': 3}
         bugs = [
-            flexmock(id='OCPBUGS-1', is_tracker_bug=lambda: True, whiteboard_component='foo'),
-            flexmock(id='OCPBUGS-2', is_tracker_bug=lambda: True, whiteboard_component='bar'),
-            flexmock(id='OCPBUGS-3', is_tracker_bug=lambda: True, whiteboard_component='buzz'),
-            flexmock(id='OCPBUGS-4', is_tracker_bug=lambda: False),
-            flexmock(id='OCPBUGS-5', is_tracker_bug=lambda: False)
+            flexmock(id='OCPBUGS-1', is_tracker_bug=lambda: True, is_cve_in_summary=lambda: True, whiteboard_component='foo'),
```

Comment:
Think this is a sign that the summary is not initialized properly. I think the property should return an empty string if that jira field is not set. Then it is not necessary to patch this method all over the place and distract from what is actually being tested.

Reply:
verify function is added to validate, so bugs require this method is appropriate

Reply:
I am not sure I follow. 
If the summary is properly initialized, there is no need to patch this function.
I think the option of initializing the summary in the class is better than needing to patch this function in any unit test that run that touch that function.